### ISSUE TITLE
Page does not load when used via proxy/domain name

### ISSUE THREAD

**Issue by Maypul** [usememos/memos#3124]
### Describe the bug

Hello! I have provisioned Memos via docker and it's behind proxy, Traefik.
I have searched the github and it is not an issue with Firefox. I have testes latest/test/stable tags. I get a while blank page in all browsers I have.
Hovewer, if I map port directly and access it via local IP, the page loads just fine.
If I try domain name via Traefik I always get blank page, no error logs in container.

Here's my compose:

  memos:
    container_name: Memos
    hostname: memos
    image: neosmemo/memos:stable
    ports:
      - 5230:5230 #this works
    volumes:
      - $SHR_DOCKER/Memos:/var/opt/memos:rw
    environment:
      MEMOS_DRIVER: postgres
      MEMOS_DSN: $MEMOPASS
    labels:
      traefik.enable: true
      traefik.http.routers.memos-rtr.entrypoints: https
      traefik.http.routers.memos-rtr.rule: Host(`memos.example.com`)
      traefik.http.routers.memos-rtr.tls: true
      traefik.http.routers.memos-rtr.middlewares: authentik-login@file
      traefik.http.routers.memos-rtr.service: memos-svc
      traefik.http.services.memos-svc.loadbalancer.server.port: 5230 #this gets me blank page.

### Steps to reproduce

Provision Memos instance on docker
Use Traefik to access it via domain name (compose provided)
See blank page if accessed via domain name

### The version of Memos you're using

Stable (testes also test and latest tags)

### Screenshots or additional context

https://prnt.sc/aKPwAqm-tbX1 (blank page with F12 logs)

**Comment by RaphaelPour** [comment]
My [caddy reverse-proxy works just fine with the latest version... 

Are both containers (memos and traefik) within the same docker network? Otherwise the traefik might not be able to discover the memos container and process the labels.

**Comment by Maypul** [comment]
Yes, they are both in the same network. All other containers with the same style of labels work, it is just Memos somehow does not want to load anything.

I think I have found the issue, or maybe a bug/incompability. I am unsure how I did not notice this error earlier: https://prnt.sc/QAz1iPk9f7VW

https://prnt.sc/NWy68cSD96GO - Memos basically does not like the - sign. Once I change the formatting without these, it works.
Fixed: https://prnt.sc/dEqOaOfAuk2d - though it was basic formatting for Traefik
